src/js/EditsQ029.jsx

```diff
@@ -0,0 +1,52 @@
+var React = require('react');
+var EditsDetail = require('./EditsDetail.jsx');
+
+var EditsQ029 = React.createClass({
+  propTypes: {
+    group: React.PropTypes.array
+  },
+
+  makeSelect: function() {
```

Comment:
I think the select elements should actually be radios (with no default checked). See [this article](http://uxmovement.com/forms/why-users-abandon-forms-with-select-menus/) for some usability troubles with select.